plugin/helpers_config.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package plugin
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/utils"
+)
+
+// CheckRequiredServerConfiguration checks if the server is configured according to
+// plugin requirements.
+func (p *HelpersImpl) CheckRequiredServerConfiguration(req *model.Config) (bool, error) {
+	cfg := p.API.GetConfig()
```

Comment:
nit: if `req == nil`, we don't actually need `cfg`. Can we move the check earlier?

Reply:
sure, I missed that one 👍